src/main/scala/exploration/riseExploration.scala

```diff
@@ -26,18 +28,20 @@ object riseExploration {
 
 
   // entry point for exploration
-  def apply(solution:Rise, filePath:String) = {
+  def apply(solution:Rise, filePath:String): (Rise, Option[Double]) = {
 
     // parse config file
     val parsedConfiguration = jsonParser.parse(filePath)
 
     // setup gold
     // code here
 
-    val startingPoint = prepareExploration(parsedConfiguration, solution, filePath)
+    val startingPoint = prepareExploration(parsedConfiguration,
+      solution, filePath)
 
     // start
-    startingPoint.execute(new Solution[Rise](solution, scala.collection.mutable.Seq.empty[Strategy[Rise]]))
+    startingPoint.execute(Solution(solution,
+      immutable.Seq.empty[Strategy[Rise]]))
```

Comment:
I assume changing to an immutable collection is desired?

Reply:
Yes. I don't see why the collection was immutable, to begin with.